crates/rpc/rpc-builder/src/lib.rs

```diff
@@ -725,6 +730,8 @@ pub enum RethRpcModule {
     Reth,
     /// `ots_` module
     Ots,
+    /// none
+    None,
```

Comment:
I don't think we need this,
because this is namespace level, we want selection level:

https://github.com/paradigmxyz/reth/blob/4f631cc77334d725ab680bfe79c0b166c20e1d9a/crates/rpc/rpc-builder/src/lib.rs#L678-L689

and "none" can just be `Selection(vec![])`